### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by Jack-Works** [js-choi/proposal-policy-map-set#5]
I suggest naming it like:

`Map.FIFO` `Map.LRU`

**Comment by js-choi** [comment]
Remind me—is there any precedent yet for adding constructor properties to constructors in the core language? I would like to match precedents set by the rest of the language.

**Comment by ljharb** [comment]
No, but we discussed it wrt resizable array buffers, and the precedent was for Arrays, not everything. We could set a new precedent here, and i think we should.
